docs/references/README.md

```diff
@@ -42,6 +42,19 @@ to ensure that CometBFT meets and exceeds industry benchmarks.
 
 Explore the QA documents to understand the testing methods to ensure the quality and performance of CometBFT.
 
+## Configuration Manual
+
+The [Configuration Manual](./config/README.md) is a reference manual describing all CometBFT configuration options.
+It details all configuration files deployed in the HOME folder and includes examples and best practices to ensure that
```

Comment:
```suggestion
It details all configuration files deployed in the CMTHOME folder and includes examples and best practices to ensure that
```

Reply:
Neither are fully correct, CMTHOME can be undefined. I'll just remove the reference to folders.

```suggestion
It details all configuration files deployed and includes examples and best practices to ensure that
```